<launch>

  <!-- Start CoM publisher node -->
  <node pkg="C43_LocalBodyCOM" type="computeCOM" name="c43_local_body"/>
 
  <!-- zmp_node  -->
  <node pkg="C42_ZMPWalk" type="zmp_main.py" name="zmp_movement_plan" />
  
  <!-- Leg IK caller -->
  <node pkg="C42_Leg_IK" type="IK_Caller.py" name="IK_Caller" />

 </launch>
